service/controller/internal/vmsku/vmsku.go

```diff
@@ -0,0 +1,107 @@
+package vmsku
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/azure-operator/v4/client"
+)
+
+const (
+	// CapabilitySupported is the value returned by this API from Azure when the capability is supported
+	CapabilitySupported = "True"
+
+	CapabilityAcceleratedNetworking = "AcceleratedNetworkingEnabled"
+)
+
+type Config struct {
+	ClientFactory *client.Factory
+	Location      string
+	Logger        micrologger.Logger
+}
+
+type Interface struct {
```

Comment:
Huh? Struct called `Interface`? Should you maybe define the interface in `spec.go` and then have implementation here for that? Then constructor below would be `func New(config Config) (Interface, error) {`

Reply:
Let me just rename it.